src/main/java/org/reactome/addlinks/fileprocessors/VGNCFileProcessor.java

```diff
@@ -0,0 +1,64 @@
+package org.reactome.addlinks.fileprocessors;
+
+import org.reactome.addlinks.EnsemblBiomartUtil;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class VGNCFileProcessor extends FileProcessor{
+
+    public VGNCFileProcessor(String processorName)
+    {
+        super(processorName);
+    }
+
+    public VGNCFileProcessor()
+    {
+        super();
+    }
+
+    private static final int vgncIdentifierIndex = 1;
+    private static final int uniprotIdentifiersIndex = 21;
+
+    /**
+     * Build map of UniProt identifiers to Vertebrate Gene Nomenclature Committee (VGNC) identifiers that is used to create VGNC cross-references in database.
+     * @return - Map<String, List<String>
+     */
+    @Override
+    public Map<String, List<String>> getIdMappingsFromFile()
+    {
+        Map<String, List<String>> mappings = new HashMap<>();
+        try {
+            this.unzipFile(this.pathToFile, true);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
```

Comment:
Should a logger statement and either a termination of the program or the return of an empty map happen in the catch block?  The method doesn't seem like it can continue if the file doesn't get unzipped?

Reply:
Good point, I'll adjust that